docs/import/add_disaster_startup.js

```diff
@@ -22,6 +23,8 @@ const authenticator = new Authenticator(
     });
 authenticator.start();
 
+$(populateColorFunctions());
```

Comment:
I think you need to do $(populateColorFunctions), not what you have. What you have does $() around the result of populateColorFunctions(), which is executed immediately. It seems like Javascript modules are executed after page load, which I can't find documented anywhere, but confirmed experimentally. So $() is overkill, but probably good practice.

Reply:
Done.